android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsBannerAdViewManager.java

```diff
@@ -55,12 +56,12 @@
   private final String EVENT_APP_EVENT = "onAppEvent";
   private final int COMMAND_ID_RECORD_MANUAL_IMPRESSION = 1;
 
-  private AdRequest request;
-  private List<AdSize> sizes;
-  private String unitId;
-  private Boolean manualImpressionsEnabled;
-  private boolean propsChanged;
-  private boolean isFluid;
+  private final HashMap<Integer, AdRequest> requestMap = new HashMap<>();
```

Comment:
If we create a ReactAdView class that extends ReactViewGroup, we don't need maps to keep track of everything and set those props directly on the instance. See: https://github.com/sbugert/react-native-admob/blob/master/android/src/main/java/com/sbugert/rnadmob/RNAdMobBannerViewManager.java#L28